src/System/PairExchange.jl

```diff
@@ -84,11 +43,16 @@ interactions on these bonds will be overwritten. The parameter `bond` has the
 form `Bond(i, j, offset)`, where `i` and `j` are atom indices within the unit
 cell, and `offset` is a displacement in unit cells.
 
-Scalar `J` implies a pure Heisenberg exchange.
+The parameter `J` may be scalar or matrix-valued. As a convenience, `dmvec(D)`
+can be used to construct the antisymmetric part of the exchange, where `D` is
+the Dzyaloshinskii-Moriya pseudo-vector. The resulting interaction will be
+``𝐃⋅(𝐒_i×𝐒_j)``.
 
-As a convenience, `dmvec(D)` can be used to construct the antisymmetric part of
-the exchange, where `D` is the Dzyaloshinskii-Moriya pseudo-vector. The
-resulting interaction will be ``𝐃⋅(𝐒_i×𝐒_j)``.
+The optional parameter `biquad` defines the strength ``b`` for scalar
+biquadratic interactions of the form ``b (𝐒_i⋅𝐒_j)²`` For systems restricted
+to dipoles, ``b`` will be automatically renormalized for maximum consistency
+with the more variationally accurate SU(_N_) mode. This renormalization
+introduces also a correction to the quadratic part of the exchange.
```

Comment:
What you're suggesting is a general biquadratic interaction, which we don't support yet. There are currently no plans to support it in dipole mode because it would be fairly complicated (a generalization of the renormalization scheme here would need to be derived, https://arxiv.org/abs/2304.03874). The plan is to generalize this entire thing for SU(N) mode so people can build arbitrary spin polynomials, not restricted to biquadratic.